span.go

```diff
@@ -49,22 +53,35 @@ type Span interface {
 	// may ignore the tag, but shall not panic.
 	SetTag(key string, value interface{}) Span
 
-	// LogEvent() is equivalent to
+	// LogFields is an efficient and type-checked way to record key:value
+	// logging data about a Span, though the programming interface is a little
+	// more verbose than LogKV(). Here's an example:
 	//
-	//   Log(LogData{Event: event})
+	//    span.LogFields(
+	//        log.String("request_path", request.Path()),
```

Comment:
It's not supposed to replace tags, no... maybe the "request" example isn't working for you. I see this come up when people have little events during the lifetime of a Span that don't warrant their own child Spans and don't contribute to the critical path but are worth remembering.

I can choose a different sort of example (e.g., recording information about a soft error) if you'd prefer...